packages/itwinui-react/src/core/Toast/Toast.tsx

```diff
@@ -248,17 +267,34 @@ export const ToastPresentation = (props: ToastPresentationProps) => {
     hasCloseButton,
     onClose,
     className,
+    contentProps,
+    startIcon,
+    statusAreaProps,
     ...rest
   } = props;
 
   const StatusIcon = StatusIconMap[category];
 
   return (
     <Box className={cx(`iui-toast iui-${category}`, className)} {...rest}>
-      <Box className='iui-status-area'>
-        {<StatusIcon className='iui-icon' />}
+      <Box
+        {...statusAreaProps}
+        className={cx('iui-status-area', statusAreaProps?.className)}
+      >
+        {!!startIcon ? (
+          <Icon>{startIcon}</Icon>
```

Comment:
Should we apply the default `fill` according to the category even when the `startIcon` is provided? Users can then override it if they would want to.

```tsx
startIcon: (
  <SvgAirplane
    style={{
      fill: 'red',
    }}
  />
),
```

Because if they want to use the status color on a custom icon and we don't apply it by default, they'll have to handle the color for each category themselves, which is a bit difficult.

Reply:
1. Status icons with colours are standard. If user wants something custom, they have to fully customise it themselves.
2. Starting to think to remove <Icon /> wrapper and just simply render what user provided. It would be user responsibility to pass correct Icon.

Reply:
> 1. Status icons with colours are standard. If user wants something custom, they have to fully customise it themselves.

I guess both ways have benefits. My point was just that if a user just wants a custom icon but wants to keep the same fill colour, then it would be a bit of extra work to find which iui color variable to use for that category and then set it manually. Also, if they have multiple categories, then they need to know the colour for the other categories too.

> 2. Starting to think to remove  wrapper and just simply render what user provided. It would be user responsibility to pass correct Icon.

Sounds good, we can then document that we require users to wrap their icon with `<Icon>` to get some styling and accessibility features.